src/main/java/de/retest/ui/diff/IdentifyingAttributesDifferenceFinder.java

```diff
@@ -24,12 +25,8 @@ public IdentifyingAttributesDifferenceFinder( final RecheckIgnore recheckIgnore
 
 	public IdentifyingAttributesDifference differenceFor( final IdentifyingAttributes expected,
 			final IdentifyingAttributes actual ) {
-		if ( expected == null ) {
-			throw new NullPointerException( "Expected cannot be null!" );
-		}
-		if ( actual == null ) {
-			throw new NullPointerException( "Actual cannot be null!" );
-		}
+		Objects.requireNonNull( expected );
+		Objects.requireNonNull( actual );
```

Comment:
We do not need checks here, since the variables are used below. Let the NPE be implicit.